pages/1.12/security/ent/ldap/ldap-sync/index.md

```diff
@@ -16,7 +16,7 @@ In earlier versions of DC/OS Enterprise LDAP Group Import and LDAP User Import w
 
 In DC/OS Enterprise v1.12 we added automatic LDAP Synchronization. This feature is enabled by default and periodically performs LDAP Synchronization. This feature is designed to keep IAM users, IAM user groups and the relations between them synchronized with their counterparts in the external Directory.
 
-**A quick note on security:** LDAP Synchronization occurs periodically. The exact period is configurable (See [iam_ldap_sync_interval config]()). There are cases where you modify the external Directory, e.g. to perform a permission revocation through modification of group membership or removal of user accounts. In some cases, these changes must be reflected in the DC/OS IAM within seconds, not minutes and therefore waiting for the next LDAP Synchronization event is not an option. For these scenarios you will have to make the equivalent changes to the DC/OS IAM instead of waiting for the next LDAP Synchronization event.
+**A quick note on security:** LDAP Synchronization occurs periodically. There are cases where you modify the external Directory, e.g. to perform a permission revocation through modification of group membership or removal of user accounts. In some cases, these changes must be reflected in the DC/OS IAM within seconds, not minutes and therefore waiting for the next LDAP Synchronization event is not an option. For these scenarios you will have to make the equivalent changes to the DC/OS IAM instead of waiting for the next LDAP Synchronization event.
```

Comment:
Why is the Synchronization word with capital S?

Reply:
Both Synchronization and Directory were treated as proper nouns, both are now corrected to lower case, as well as one more pass for readability.